### ISSUE TITLE
Error on PackerCompile

### ISSUE THREAD

**Issue by Nesty92** [wbthomason/packer.nvim#561]
- `nvim --version`:  v0.5.0-dev+1372-g056c464e8
- `git --version`: 2.32.0
- Operating system/version: Manjaro Linux 21.1.0
- Terminal name/version: yakuake 21.04.3

### Steps to reproduce
clone https://github.com/NvChad/NvChad and run PackerCompile
### packer files

<details>
<summary>Plugin specification file(s)</summary>

Post or link your plugin specification files here, if you aren't able to provide a minimal
reproducer

</details>

<details>
<summary>packer log file</summary>

Post the contents of ~/.cache/nvim/packer.nvim.log here
```
[ERROR mié 18 ago 2021 12:03:52 3493810429949] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[ERROR mié 18 ago 2021 17:38:45 23586127078221] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[ERROR mié 18 ago 2021 17:42:12 23793223173769] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[ERROR mié 18 ago 2021 17:45:04 23965702839116] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[ERROR mié 18 ago 2021 17:46:33 24053937918553] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[ERROR mié 18 ago 2021 17:47:03 24084201549505] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[ERROR mié 18 ago 2021 17:50:38 24299356574916] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
[WARN  mié 18 ago 2021 18:01:50 24970948637588] ...im/site/pack/packer/opt/packer.nvim/lua/packer/clean.lua:79: Cleaning cancelled!
[ERROR mié 18 ago 2021 18:01:52 24973101496497] ...im/site/pack/packer/opt/packer.nvim/lua/packer/async.lua:20: Error in coroutine: ...are/nvim/site/pack/packer/opt/packer.nvim/lua/packer.lua:698: Vim:E492: Not an editor command: -- Automatically generated packer.nvim plugin loader code
```
</details>

<details>
<summary>packer compiled file</summary>

Post the contents of `packer_compiled.vim` here

</details>

**Comment by akinsho** [comment]
Hi @Nesty92, can you provide a more minimal example to reproduce this issue since NvChad is an entire neovim configuration so debugging this issue would involve delving into that repo's setup, which makes this much more time-consuming to help with. Can you either raise the issue there or narrow down the issue into a *minimal* `init.lua` that can then be used to reproduce this issue.

**Comment by jasmcaus** [comment]
I'm not sure about @Nesty92, but I've run into some `packer.nvim` issues. I too am using NVChad, and everything was working fine until today. My `plugin/packer_compiled.lua` file was deleted, so I attempted to run `:PackerCompile`, but received the following error instead:

![image](https://user-images.githubusercontent.com/61184127/130348603-8750373a-3ff3-406c-9a0b-61563e251843.png)

I haven't updated any of my configurations since yesterday (when it was working fine) -- the only change was that I had deleted the `plugin/packer_compiled.lua` file accidentally.
